### PR TITLE
fix(githubactions): github actions relative path detected as not pinned

### LINKED ISSUE #6957: bug(github_actions): github actions relative path detected as not pinned
### Expected Behavior

KICS should not detect rule "Unpinned Actions Full Length Commit SHA" when a relative path is used in the `uses` field in a Github Actions workflow.

### Actual Behavior

The "Action is not pinned to a full length commit SHA." rule (555ab8f9-2001-455e-a077-f2d0f41e2fb9) shows as failed for action at step `relative-path`, which has a relative path in the same repository.

### Steps to Reproduce the Problem

1.  Create a minimal github action (`.github/workflows/sample.yml`):
```
name: Sample
on:
  workflow_call:
permissions: read-all
jobs:
  sample:
    name: Sample
    runs-on: ubuntu-latest
    steps:
      - name: Relative path action
        id: relative-path
        uses: ./any-action
```
2.  Run KICS `docker run -t -v $(pwd)/.github:/path checkmarx/kics:latest scan -p /path -o "/path/" --report-formats "json,sarif,html"`
3.  Look at the findings, a finding is listed as described above

### Specifications
(N/A if not applicable)

-   Version: [v1.7.13](https://github.com/Checkmarx/kics/releases/tag/v1.7.13), through the `latest` tag in Docker
-   Platform: macOS 13.6 (22G120)
-   Subsystem: N/A

### PR DESCRIPTION
**Pull request by cw-alexcroteau**
Closes #6957

**Proposed Changes**
- Add an exception for relative paths for rule "Action is not pinned to a full length commit SHA." (555ab8f9-2001-455e-a077-f2d0f41e2fb9)

I submit this contribution under the Apache-2.0 license.

### CODE REVIEW

File: assets/queries/cicd/github/unpinned_actions_full_length_commit_sha/query.rego [hunk 1]
```diff
@@ -28,3 +29,8 @@ isPinned(use){
 	regex.match("@[a-f0-9]{40}$", use)
 }
 
+isRelative(use){
+	allowed := ["./", "../"]
```
Comment (JoaoCxMartins) [inline assets/queries/cicd/github/unpinned_actions_full_length_commit_sha/query.rego:33]: Hi @cw-alexcroteau.
First, thank you for your contribution :).
I'm with one doubt, relative to the "../".
Why do you feel the necessity to support the "../" ?

**Comment by JoaoCxMartins** [comment]
Hi @cw-alexcroteau ,
how are you ? 
Can you sync your fork please ?
Br,
João Martins

File: assets/queries/cicd/github/unpinned_actions_full_length_commit_sha/query.rego (hunk 1 above)
[the remark this answers is not in the document]
Comment (cw-alexcroteau) [reply assets/queries/cicd/github/unpinned_actions_full_length_commit_sha/query.rego:33]: Hi, you are right, no need to support it. I completely missed your reply and carried on, let me remove that and sync my fork.

**Comment by cw-alexcroteau** [comment]
> Hi @cw-alexcroteau , how are you ? Can you sync your fork please ? Br, João Martins

Hi @JoaoCxMartins, thanks for the heads up, it's now synced.

**Review by JoaoCxMartins — APPROVED** [review]

**Comment by JoaoCxMartins** [comment]
Hi @cw-alexcroteau TY for your contribution to kicks

**Pull request merged**